### ISSUE TITLE
Add h2 license

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode-toolkit#995]
Reported privately
See https://repository.jboss.org/licenses/h2.txt
This is very close to https://opensource.org/licenses/cuaoffice.php and based on an MPL of sorts
It also has this exhibit:
```
Exhibit A

Multiple-Licensed under the H2 License, Version 1.0,
and under the Eclipse Public License, Version 1.0
(http://h2database.com/html/license.html).
Initial Developer: H2 Group
```

**Comment by armijnhemel** [comment]
same as #994

**Comment by pombredanne** [comment]
Closed in favor of #994
